src/store/modules/layers.store.js

```diff
@@ -465,7 +466,7 @@ const actions = {
             if (!extent) {
                 updatedLayer.errorKey = 'kml_gpx_file_empty'
                 updatedLayer.hasError = true
-            } else if (!getKmlExtentForProjection(rootState.position.projection, extent)) {
+            } else if (!getExtentForProjection(rootState.position.projection, extent)) {
```

Comment:
I've just seen that in your previous PR for GPX support it seems that you don't support GPX after a reload as the GPX data will not be set (no store subscriber to get the GPX data from url). This means that you would need to add that support and also need a similar logic in the layer store as here and have a `updateGpxLayer()` and test here again for the extent being in projection bound like for kml. So we would handle the case were an online GPX is updated and the new version is out of bound.

Reply:
That's something I wanted to do in another PR